bindings_ffi/src/mls.rs

```diff
@@ -216,11 +216,11 @@ impl FfiConversations {
         if !account_addresses.is_empty() {
             convo.add_members(account_addresses).await?;
         }
-
         let out = Arc::new(FfiGroup {
             inner_client: self.inner_client.clone(),
             group_id: convo.group_id,
             created_at_ns: convo.created_at_ns,
+            added_by_address: Some(self.inner_client.account_address()),
```

Comment:
We should read this from `convo` so that the value is guaranteed to be consistent with `convo`